src/_data/homepage_blueprints.yaml

```diff
@@ -0,0 +1,15 @@
+- title: "CRUD MongoDB API"
```

Comment:
I would assume that this data file only need 3 keys, which are pointers to blueprints in the repo we already import. Why have you chosen to duplicate these values?